conda/common/configuration.py

```diff
@@ -435,19 +436,19 @@ def valueflags(self, parameter_obj):
 def load_file_configs(search_path):
     # returns an ordered map of filepath and dict of raw parameter objects
 
-    def _file_yaml_loader(fullpath):
+    def _file_loader(fullpath):
```

Comment:
While strictly unnecessary this enabled me to do `find . -name '*.py' -exec grep -q yaml_load {} \; -print` and have it turn up empty, which is nice.